src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/PFD/AS1000_PFD.js

```diff
@@ -33,16 +33,27 @@ class AS1000_PFD extends BaseAS1000 {
         this.addEventLinkedPopupWindow(new NavSystemEventLinkedPopUpWindow("Procedures", "ProceduresWindow", new MFD_Procedures(), "PROC_Push"));
         this.addEventLinkedPopupWindow(new NavSystemEventLinkedPopUpWindow("CONFIG", "PfdConfWindow", new AS1000_PFD_ConfigMenu(), "CONF_MENU_Push"));
         this.maxUpdateBudget = 12;
-        let avionicsKnobIndex = 30;
-        let avionicsKnobValue = SimVar.GetSimVarValue("A:LIGHT POTENTIOMETER:" + this.avionicsKnobIndex, "number");
+        this.avionicsKnobIndex = 30;
+        this.avionicsKnobValue = 0; // SimVar.GetSimVarValue("A:LIGHT POTENTIOMETER:" + this.avionicsKnobIndex, "number");
+        this._interiorCfgLoaded = false;
```

Comment:
Could we make a `AS1000Config` class that attaches here instead that holds this state and talks to the config backend?

Reply:
1)   I need 'line' in the loop there so I can't use empty params.   I tried using _ for the throwaways which I was pretty sure _shoud_ work, but it didn't.

2) I was thinking of throwing `value = value.toLowerCase()` in this block for just that reason.  I'll go ahead and add that.

3) For sure, this section needs a heavy rewrite with @tavip's recent work merged in.   I plan on doing just that, creating a config class which pulls from simvars or the data storage and provides a cleaner interface to the mian code.